services/app-api/src/parameterStore/emailParameterStore/helpers.ts

```diff
@@ -0,0 +1,27 @@
+import { ParameterStoreType } from '../awsParameterStore'
+
+const validateAndReturnValueArray = (
+    storeResponse: ParameterStoreType,
+    name: string
+): Error | string[] => {
+    if (storeResponse instanceof Error) {
+        return storeResponse
+    }
+
+    const { type, value } = storeResponse
+    if (type !== 'StringList') {
+        const errorMessage = `Parameter store ${name} value of Type ${type} is not supported`
+        return new Error(errorMessage)
+    }
+
+    //Split string into array using ',' separator and trim each array item.
+    const stringArray = value.split(',').map((email) => email.trim())
+
+    if (stringArray.length === 0) {
```

Comment:
Just pulling out that block is new logic I added yesterday. It would error already on apollo_gql where parameter store is called if the parameter is somehow empty. Before we only bailed out if it was empty or wrong type. Eventually we can use this kind of  fucntion for every parameter I just started with divisions.